emsdk_manifest.json

```diff
@@ -540,6 +569,12 @@
   ],
 
   "sdks": [
+  {
+    "version": "wasm-git-master",
```

Comment:
I think we've been calling this "upstream" as opposed to "fastcomp".  Would that make more sense that "wasm" here?

Reply:
How about "upstream-incoming"?

Reply:
Or perhaps call this one "upstream" and rename the below one to `fastcomp-incoming` to match what we did for `latest` vs `fastcomp`?

Reply:
I was thinking the whole word `upstream` will begin to mean less once `fastcomp` becomes more and more obsolete, so called this just "wasm from git, master branch". But sure, I'll rename to upstream-incoming.

The rename happens in https://github.com/emscripten-core/emsdk/pull/400, don't want to mix that here.

Reply:
I'm fine with the word upstream disappearing, I just rather not replace with with another term such as "wasm" that we haven't been using to refer to llvm before.